MathPortal/MathPortal/Source/Custom/Custom Views/SolutionView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  SolutionView.swift
+//  MathPortal
+//
+//  Created by Petra Čačkov on 03/10/2019.
+//  Copyright © 2019 Petra Čačkov. All rights reserved.
+//
+
+import UIKit
+
+class SolutionView: UIView {
+    
+    var solution: TaskSolution?
+    private var equationViewsHeight: CGFloat = 0
+    private var ownerInfoViewHeight: CGFloat = 60
+    // NOTE: 2 is just a placeholder since cell already has width
+    override var intrinsicContentSize: CGSize {
+        return CGSize(width: 2, height: equationViewsHeight + ownerInfoViewHeight)
+    }
+    
+    func setupView(solution: TaskSolution) {
```

Comment:
Since you have this method it might make sense to have solution `private(set)`. Or does it still make sense to modify it without calling this method?